docs/en/ingest-management/fleet/monitor-elastic-agent.asciidoc

```diff
@@ -178,7 +178,7 @@ image::images/collect-agent-diagnostics.png[Collect agent diagnostics under agen
 
 Any in-progress or previously collected bundles for the {agent} will be listed on this page.
 
-Note that the bundles are stored in {es} and will be removed after 30 days.
+Note that the bundles are stored in {es} and will be removed after 30 days. You can also delete any previously created bundle by clicking the `trash can` icon.
```

Comment:
actually the [default ILM policy](https://github.com/elastic/elasticsearch/blob/5a52642db705b450ab14efa86c781462516711d5/x-pack/plugin/core/template-resources/src/main/resources/fleet-file-fromhost-data-ilm-policy.json#L13) for where these diagnostics files are stored deletes them after 7 days, this wasn't changed recently so not sure where the original info came from 😅

Reply:
Thanks! Fixed in [2911cb9](https://github.com/elastic/ingest-docs/pull/1081/commits/2911cb925942d0b9df02138ac5cd7455c65857d2).